### DISCUSSION TITLE
Should I convert data after moving to 0.2.5?

### DISCUSSION THREAD

**Discussion by wsw70** [zinclabs/zinc#238] [Q&A]
During the transition between 0.2.4 and 0.2.5 there was a change in the data format. [Thanks to](https://github.com/zinclabs/zinc/issues/233) @hengfeiyang I added the lines below to my docker definition and I have access again:

```text
ZINC_METADATA_STORAGE: badger
ZINC_ICE_COMPRESSOR: s2
```

This got me wondering whether I should convert the data somehow? (and remove the extra env lines afterward)?

**Comment by hengfeiyang** [comment]
No need for it. We just add the feature let user can custom `@timestamp`. Not effect default action.

**Reply by wsw70** [reply]
I am not sure I understand: I was [already using](https://github.com/zinclabs/zinc/discussions/229#discussioncomment-3021710) a custom `@timestamp` in 0.2.4. It seems to me that that part of the code was not changed (?).

I had a look at the code and I do not see how the lines 
```
ZINC_METADATA_STORAGE: badger
ZINC_ICE_COMPRESSOR: s2
```
are related to `@timestamp` ?

**Reply by hengfeiyang** [reply]
Yes, In v0.2.4 just support standard format of timestamp, In v0.2.5 we support format and time_zone for custom `@timestamp`.

**Reply by hengfeiyang** [reply]
```
ZINC_METADATA_STORAGE: badger
ZINC_ICE_COMPRESSOR: s2
```

This not related timestamp, this related performance improve.

In v0.2.5, we use `ZINC_METADATA_STORAGE=bolt` as default storage, it will reduce CPU usage.

Also `ZINC_ICE_COMPRESSOR=s2` (default in v0.2.4) will reduce CPU usage but has low compress ratio.

Use `ZINC_ICE_COMPRESSOR=zstd` (default is v0.2.5) will has high compress ratio but higher CPU usage.
